### PR TITLE
[16.0][MIG] endpoint

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Standard migration

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
cc @EvaSForgeFlow

**Review by OriolMForgeFlow — APPROVED** [review]
LGTM!

File: endpoint/hooks.py [hunk 1]
```diff
@@ -1,76 +0,0 @@
...
-
-_logger = logging.getLogger(__file__)
-
-
-def post_init_hook(cr, registry):
-    env = api.Environment(cr, SUPERUSER_ID, {})
-    _init_server_action(env)
-
-
-def _init_server_action(env):
```
Comment (simahawk) [inline endpoint/hooks.py]: why trashing this?

**Review by simahawk — CHANGES_REQUESTED** [review]

File: endpoint/hooks.py (hunk 1 above)
Comment (OriolMForgeFlow) [reply endpoint/hooks.py]: In version 16, the field 'activity_user_type' is no longer required=True. It has been changed to be computed. 
As a result, the error `psycopg2.IntegrityError: null value in column "activity_user_type" violates the not-null constraint`, which was previously fixed by the hook in previous versions, is no longer appearing.

Comment (simahawk) [reply endpoint/hooks.py]: Ok, but where is the server action then? :)

Comment (OriolMForgeFlow) [reply endpoint/hooks.py]: You are right, now it is loaded from a data file.

**Review by simahawk — APPROVED** [review]

**Pull request merged**